src/coreclr/nativeaot/Runtime/RhConfigValues.h

```diff
@@ -16,6 +16,7 @@ DEBUG_CONFIG_VALUE_WITH_DEFAULT(BreakOnAssert, 1)
 RETAIL_CONFIG_VALUE(StressLogLevel)
 RETAIL_CONFIG_VALUE(TotalStressLogSize)
 RETAIL_CONFIG_VALUE(gcServer)
+RETAIL_CONFIG_VALUE(GcConservativeStacks)   // Enables conservative stack reporting
```

Comment:
We should call it the same thing as in CoreCLR (https://github.com/dotnet/runtime/blob/242c95aab1085dc5af9a017c00fe78c58386229f/src/coreclr/inc/clrconfigvalues.h#L268)

Reply:
`gcConservative` turns support for conservative reporting in the GC. Native AOT enables that unconditionally.
I think if we use the same key we will see it always "on".

Reply:
`gcConservative` turns on conservative stack scanning in CoreCLR: https://github.com/dotnet/runtime/blob/4cf2e3bb18162d750e7a128bb805c49c9fd52612/src/coreclr/vm/gcenv.ee.cpp#L129. 

> NativeAOT enables that unconditionally. I think if we use the same key we will see it always "on".

The GC should see it as always on in NativeAOT, but the runtime should use the env variable that you are adding.

Reply:
Actually it looks like we intercept `gcConservative` when GC asks about it. We may be able to use the same key.

Another issue could be that `gcConservative` would affect the test infrastructure as well as NativeAOT tests. That is probably ok.